packages/millicast-sdk/src/utils/Diagnostics.js

```diff
@@ -11,6 +11,7 @@ const diagnostics = {
   connection: '',
   stats: []
 }
+const tempStats = []
```

Comment:
Please refactor the diagnostics object, as we discussed, some information are constants such as version, accountId, streamName, subscriberID, userAgent.   Since we return the copy of `tempStats`, we could just name it `stats`